docs/api/puppeteer.autofilldata.md

```diff
@@ -14,4 +14,4 @@ export interface AutofillData
 
 | Property   | Modifiers | Type                                                                                    | Description | Default |
 | ---------- | --------- | --------------------------------------------------------------------------------------- | ----------- | ------- |
-| creditCard |           | { number: string; name: string; expiryMonth: string; expiryYear: string; cvc: string; } |             |         |
```

Comment:
do you know what caused changes in the markdown documentation?

Reply:
I needed to add the escape on the brackets according to the migration documentation attached to the issue otherwise MDX V3 was throwing an error

Reply:
The md files are auto-generated though (npm run docs). I guess we need to fix the api docs generator too?

Reply:
Yes, updated by adding the html code for the curly brackets